packages/pointer_interceptor/pointer_interceptor_platform_interface/AUTHORS

```diff
@@ -0,0 +1,6 @@
+# Below is a list of people and organizations that have contributed
+# to the project. Names should be added to the list like so:
+#
+#   Name/Organization <email address>
```

Comment:
probably wanna put your name there?

Reply:
This is the format example.

The actual entries are below, and Google-employed contributors are covered by "Google".